.github/workflows/tests.yml

```diff
@@ -85,8 +85,9 @@ jobs:
     strategy:
       fail-fast: true
       matrix:
-        php: [8.2, 8.3]
+        php: [8.3]
```

Comment:
The `mysql` job matrix covers all of the supported PHP versions so there didn't seem to be much pointing in running all of the versions against Postgres and SQLite.